src/components/Book.jsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import BookstoreAPI from '../services/BookstoreAPI';
+
+const Book = ({
+  id, category, title, author, currentChapter, totalChapters,
+}) => {
+  const dispatch = useDispatch();
+  const handleClick = () => {
+    dispatch(BookstoreAPI.deleteBookFromAPI(id));
+    dispatch(BookstoreAPI.getAllBooksFromAPI());
+  };
```

Comment:
- [ ] As stated in the project requirement, you are meant to be adding and removing books from the local redux store and not the API, I'll suggest you update your code to dispatch actions `addBook, removeBook` you created in the redux store. :+1: :+1: